src/Web/Masa.Auth.Web.Admin.Rcl/Pages/DefaultIcon.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Stack.Components;
+
+public class DefaultIcon : MIcon
```

Comment:
Duplicated DefualtIcon?

Reply:
This component has been removed in the new branch